code/modules/food&drinks/food/snacks/meat.dm

```diff
@@ -74,6 +74,32 @@
 	desc = "Delicious dino damage"
 	filling_color = "#6B8E23"
 
+/obj/item/weapon/reagent_containers/food/snacks/meat/slab/human/mutant/lizard
```

Comment:
Pratically pretty much all the meats are defined twice here, with different variables set. Why don't you just do a /obj/item/weapon/reagent_containers/food/snacks/meat/slab/human/mutant/lizard and then define all the vars in it instead of defining it twice?It may not cause issue ingame but it sure as hell is ugly codewise